environments/perftest.yml

```
@@ -20,3 +20,6 @@ cname:
   - name: "benefit-appeal"
     ttl: 3600
     record: "hmcts-perftest.azurefd.net"
+  - name: "bulkscan"
+    ttl: 3600
+    record: "ad037a62-5f44-4add-9862-ecf0f1d6e7f8.cloudapp.net"
```

> The custom domain name could not be verified. CNAME mapping from bulkscan.perftest.platform.hmcts.net to any of bulkscanperftest.blob.core.windows.net,bulkscanperftest.z33.web.core.windows.net does not exist.

you can change it to the AG after you've validated the domain:

```suggestion
    record: "bulkscanperftest.blob.core.windows.net"
```